config/locales/referee_interface/referee_interface.yml

```
@@ -60,15 +60,15 @@ en:
             relationship_confirmation:
               blank: Choose if the described relationship is correct
             relationship_correction:
-              blank: "Enter your relationship to %{candidate}"
+              blank: "Enter how you know %{candidate}"
               too_many_words: Your description must be %{count} words or fewer
         referee_interface/reference_safeguarding_form:
           attributes:
             any_safeguarding_concerns:
-              blank: Select if you know of any reason why %{candidate} should not work with children
+              blank: Select if you have any concerns about %{candidate} working with children
             safeguarding_concerns:
-              blank: Enter a reason why %{candidate} should not work with children
-              too_many_words: Your reason must be %{count} words or fewer
+              blank: Enter your concerns about %{candidate} working with children
+              too_many_words: Your concerns must be %{count} words or fewer
```

The Design System guidance has changed to recommend using "less" instead of "fewer".

```suggestion
              too_many_words: Your concerns must be %{count} words or less
```